sensor_msgs/msg/BatteryState.msg

```diff
@@ -0,0 +1,43 @@
+# Power supply status parallel to defines in power_supply.h
```

Comment:
Should probably specify that the `power_supply.h` header is from the linux sourcecode, since some people may not assume that. Might also be worth specifying the version of the linux sourcecode it has been borrowed from.